packages/components/src/horizontal-bar-list/_stats-variables.scss

```diff
@@ -17,3 +17,16 @@ $header-font-size: 20px;
 $header-line-height: 26px;
 $stats-avatar-size: 20px;
 $stats-avatar-spacer: 8px;
+
+@mixin additional-columns-wrapper() {
+	display: flex;
+	text-align: right;
+	gap: 0 4px;
+	padding: 0 4px;
+	font-weight: 400;
+
+	> div,
+	> span {
+		min-width: 120px;
```

Comment:
I'm not sure how to avoid adding this value due to the need of aligning all columns and maintaining ellipsis for long titles. If showing more columns (with this component) will be used in any other place this will have to be moved to inline styles.